vm/actor/src/builtin/market/mod.rs

```diff
@@ -86,9 +81,42 @@ impl Actor {
         rt.create(&st)?;
         Ok(())
     }
+
```

Comment:
What's the reason for moving these market methods around?

Reply:
They weren't in order, it was bugging me a bit to follow some of them lol, can swap back if you guys would like

Reply:
No, not a problem. I just was curious if it was required.

Reply:
Yeah no functional change there, mb should have just noticed when it was PRed or not changed again but yeah just easier to jump to implementations